app/forms/my/look_and_feel_form.rb

```diff
@@ -38,14 +38,31 @@ class My::LookAndFeelForm < ApplicationForm
       caption: I18n.t("activerecord.attributes.user_preference.mode_guideline"),
       required: true,
       include_blank: false,
-      input_width: :small
+      input_width: :small,
+      data: {
+        my__look_and_feel_target: "themeSelect",
+        action: "my--look-and-feel#updateContrastOptions"
+      }
     ) do |select|
-      theme_options_for_select.each do |theme|
-        select.option(
-          value: theme[1],
-          label: theme[0]
-        )
-      end
+      theme_options_for_select.each { |(label, value)| select.option(value:, label:) }
+    end
+
+    f.check_box_group(data: { my__look_and_feel_target: "autoThemeContrast" }) do |group|
+      group.check_box name: :enable_auto_light_theme_contrast,
+                      label: I18n.t("activerecord.attributes.user_preference.enable_auto_light_theme_contrast"),
```

Comment:
```suggestion
                      label: UserPreference.human_attribute_name(:enable_auto_light_theme_contrast),
```

or if you override `model`:

```suggestion
                      label: attribute_name(:enable_auto_light_theme_contrast),
```